core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -915,3 +915,33 @@ private class RDDPartitionSeq extends Seq[v1.RDDPartitionInfo] {
   }
 
 }
+
+private[spark] class LiveWorkers(val workerId: String, _addTime: Long) extends LiveEntity {
```

Comment:
similarly throughout this code, I don't want to reference to these as Workers because that will get confusing with standalone worker

Reply:
Refactor the Worker with Process in the code change